src/css/brand-colors.css

```diff
@@ -0,0 +1,73 @@
+/**
+ * @deprecated these brand colors have been deprecated in favor of the new brand colors
+ * Brand Colors
+ * Do not use "--brand-colors" in your code
+ * Instead use the "--color-" variables to ensure
+ * theme compatible styles
+ */
+:root {
+  --brand-colors-white-white000: #ffffff;
+  --brand-colors-white-white010: #fcfcfc;
+  --brand-colors-black-black000: #000000;
+  --brand-colors-grey-grey030: #fafbfc;
+  --brand-colors-grey-grey040: #f2f4f6;
+  --brand-colors-grey-grey100: #d6d9dc;
+  --brand-colors-grey-grey200: #bbc0c5;
+  --brand-colors-grey-grey300: #9fa6ae;
+  --brand-colors-grey-grey400: #848c96;
+  --brand-colors-grey-grey500: #6a737d;
+  --brand-colors-grey-grey600: #535a61;
+  --brand-colors-grey-grey700: #3b4046;
+  --brand-colors-grey-grey750: #2e3339;
+  --brand-colors-grey-grey800: #24272a;
+  --brand-colors-grey-grey900: #141618;
+  --brand-colors-blue-blue000: #eaf6ff;
+  --brand-colors-blue-blue100: #a7d9fe;
+  --brand-colors-blue-blue200: #75c4fd;
+  --brand-colors-blue-blue300: #43aefc;
+  --brand-colors-blue-blue400: #1098fc;
+  --brand-colors-blue-blue500: #0376c9;
+  --brand-colors-blue-blue600: #0260a4;
+  --brand-colors-blue-blue700: #024272;
+  --brand-colors-blue-blue800: #01253f;
+  --brand-colors-blue-blue900: #00080d;
+  --brand-colors-orange-orange000: #fef5ef;
+  --brand-colors-orange-orange100: #fde2cf;
+  --brand-colors-orange-orange200: #fbc49d;
+  --brand-colors-orange-orange300: #faa66c;
+  --brand-colors-orange-orange400: #f8883b;
+  --brand-colors-orange-orange500: #f66a0a;
+  --brand-colors-orange-orange600: #bf5208;
+  --brand-colors-orange-orange700: #954005;
+  --brand-colors-orange-orange800: #632b04;
+  --brand-colors-orange-orange900: #321602;
+  --brand-colors-green-green000: #f3fcf5;
+  --brand-colors-green-green100: #d6ffdf;
+  --brand-colors-green-green200: #afecbd;
+  --brand-colors-green-green300: #86e29b;
+  --brand-colors-green-green400: #5dd879;
+  --brand-colors-green-green500: #28a745;
+  --brand-colors-green-green600: #1c8234;
+  --brand-colors-green-green700: #145523;
+  --brand-colors-green-green800: #0a2c12;
+  --brand-colors-green-green900: #041007;
+  --brand-colors-red-red000: #fcf2f3;
+  --brand-colors-red-red100: #f7d5d8;
+  --brand-colors-red-red200: #f1b9be;
+  --brand-colors-red-red300: #e88f97;
+  --brand-colors-red-red400: #e06470;
+  --brand-colors-red-red500: #d73847;
+  --brand-colors-red-red600: #b92534;
+  --brand-colors-red-red700: #8e1d28;
+  --brand-colors-red-red800: #64141c;
+  --brand-colors-red-red900: #3a0c10;
+  --brand-colors-purple-purple500: #8b45b6;
+  --brand-colors-violet-violet300: #cfb5f0;
+  --brand-colors-yellow-yellow000: #fffdf8;
+  --brand-colors-yellow-yellow100: #fefcde;
+  --brand-colors-yellow-yellow200: #fff2c5;
+  --brand-colors-yellow-yellow300: #ffeaa3;
+  --brand-colors-yellow-yellow400: #ffdf70;
+  --brand-colors-yellow-yellow500: #ffd33d;
+  --brand-colors-yellow-yellow600: #ffc70a;
+}
```

Comment:
Separating brand colors out into a separate stylesheet. This will give us the oppurtunity to create a `brand-colors-deprecated.css` stylesheet in the next major release of the design tokens